modules/multy-s3-bucket/0.1.0/s3-backup-bucket.tf

```diff
@@ -7,12 +7,6 @@ output "replica_s3_bucket_arn" {
   value = aws_s3_bucket.replica.arn
 }
 
-resource "aws_s3_bucket_acl" "replica" {
```

Comment:
This was throwing an error in terraform cloud for loki in particular:

```
Error: error creating S3 bucket ACL for glueops-tenant-venkata-test-33-np-loki-replica: AccessControlListNotSupported: The bucket does not allow ACLs status code: 400, request id: ESDBC9K0FF7HHK7A, host id: Ba4pUzn0L0US7SjTaVqDQR2T1pyOMtFRoEPgvU8JVBt3h+3FgLRxTEP+zbBQmVCOA+6U0IPYR+8=
with module.tenant_venkata.module.loki_s3["test-33-np"].aws_s3_bucket_acl.replica
on .terraform/modules/tenant_venkata/modules/multy-s3-bucket/0.1.0/s3-backup-bucket.tf line 10, in resource "aws_s3_bucket_acl" "replica":
```

This might be temporary bug with the S3 region but since we already have block public access enabled here: https://github.com/GlueOps/terraform-module-cloud-multy-prerequisites/pull/19/files#diff-1f3f17d185e46ffa935b2589a15e73db07d5071745e8affe461e4a57cef8ad29R52 I went ahead and removed it on the replica bucket.

Reply:
interesting - I haven't run in to that before